src/components/molecules/TodoActions.vue

```diff
@@ -2,44 +2,80 @@
 import { computed } from 'vue'
 import AppButton from '@/components/atoms/AppButton.vue'
 import { useStore } from 'vuex'
+import { FILTER_LABELS } from '@/constants/constants'
 
 const store = useStore()
 
 const filters = computed(() => store.getters.getFilters)
 const remaining = computed(() => store.getters.undoneTodosCount)
+const activeFilter = computed(() => store.getters.getActiveFilter)
 const clearCompleted = () => store.dispatch('clearCompleted')
 const changeVisibility = (visibility) => store.dispatch('changeVisibility', visibility)
 
-const capitalize = (s) => s.charAt(0).toUpperCase() + s.slice(1)
+const remainingText = computed(() => {
+  const value = remaining.value
+
+  switch (value) {
+    case 0:
+      return 'No items left'
+    case 1:
+      return '1 item left'
+    default:
+      return value + ' items left'
+  }
+})
 </script>
 
 <template>
-  <div class="task-actions">
-    <span>{{ remaining }} items left</span>
-    <ul>
-      <AppButton
-        v-for="(val, key) in filters"
-        :key="key"
-        :label="capitalize(key)"
-        @click="changeVisibility(key)"
-        class="b"
-      />
-    </ul>
-    <AppButton label="Clear Completed" @click="clearCompleted" />
+  <div class="card-body">
+    <div class="d-flex justify-content-between align-items-center">
+      <span class="remaining">{{ remainingText }}</span>
+      <div>
+        <AppButton
+          v-for="(val, key) in filters"
```

Comment:
Could you make this loop through the values ​​from FILTERS const? I guess activeFilter will have one of the value from this constant, not from the keys of the object in the state